### DISCUSSION TITLE
2 doubts

### DISCUSSION THREAD

**Discussion by tadkn** [community/community#108901] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Good evening, I came here to the website because I came to download some files, so I had to log into my account but when I logged into my account I realized that there is no option to log in via Google and I would like to know why, as it is usually through these social networks access is faster

And my other question is why the site doesn't have a Discord server and I say that because as far as we know about the platforms, communication is much better and in fact I even searched the internet for discord and I entered the name of the site and it came up Some from what I saw topics but when clicking on them you didn't enter any server

**Comment by the-coder-o** [comment]
Woooooooo

**Comment by Ybouali** [comment]
mmmmmmmmmmmmmmmmmmmmmm
i can say to welcome in the github befor someone say welcome to you bro , in this website we store project that we created by code about the email i don't have something to say , about discord go and create one from scratch i mean create a clone for discord and store it here HHHHHHH ( i am joking bro )
